master/static/migrations/20240407235703_oss-to-ee-catch-up.tx.up.sql

```diff
@@ -0,0 +1,565 @@
+-- migrations/20200415135511_scim.tx.up.sql
+CREATE SCHEMA IF NOT EXISTS scim;
+CREATE TABLE IF NOT EXISTS scim.users (
+    id          uuid PRIMARY KEY NOT NULL,
+    user_id     INTEGER NOT NULL REFERENCES users(id),
+    external_id TEXT NULL,
+    name        jsonb NOT NULL,
+    emails      jsonb NOT NULL
+);
+
+-- migrations/20200513150510_oauth.tx.up.sql
+CREATE SCHEMA IF NOT EXISTS oauth;
+CREATE TABLE IF NOT EXISTS oauth.tokens (
+    access text NOT NULL,
+    access_create_at timestamptz NOT NULL,
+    access_expires_in bigint NOT NULL,
+    client_id text NOT NULL,
+    code text NOT NULL,
+    code_create_at timestamptz NOT NULL,
+    code_expires_in bigint NOT NULL,
+    redirect_uri text NOT NULL,
+    refresh text NOT NULL,
+    refresh_create_at timestamptz NOT NULL,
+    refresh_expires_in bigint NOT NULL,
+    scope text NOT NULL,
+    user_id text NOT NULL,
+
+    id bigserial,
```

Comment:
[We shouldn't use serial](https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_serial). Since this was pre-existing and not created by this PR, this should go on the list of tech debt we should resolve IMO.